src/item.h

```diff
@@ -2020,7 +2020,8 @@ class item : public visitable<item>
         // Place conditions that should remove fake smoke item in this sub-function
         bool process_fake_smoke( player *carrier, const tripoint &pos );
         bool process_fake_mill( player *carrier, const tripoint &pos );
-        bool process_cable( player *p, const tripoint &pos );
+        bool process_cable( player *carrier, const tripoint &pos );
```

Comment:
The clang-tidy-plugin will revert this change. It wants that parameters have the same name in declaration and in their definition.

Reply:
Good to know, it happened to me once and I was baffled what was going on. Too bad the result will be a bit inconsistent `item.h` naming scheme, but as long as it works I don't complain.

Reply:
Good to know, it happened to me once and I was baffled what was going on. Too bad the result will be a bit inconsistent item.h naming scheme, but as long as it works I don't complain.

Reply:
Good to know, it happened to me once and I was baffled what was going on. Too bad the result will be a bit inconsistent item.h naming scheme, but as long as it works I don't complain.

Reply:
I've cross-unified it.